### ISSUE TITLE
Make 32-bit distro

### ISSUE THREAD

**Issue by vitstr** [StamusNetworks/SELKS#31]
Hello!
How to make 32-bit distro SELKS?

**Comment by pevma** [comment]
You need to have all the needed 32 bit packages/OS  including suricata, elasticsearch, logstash.....
It is doable  - I suspect it will be time consuming.

**Comment by yehias** [comment]
I don't recommend selks on 32 bit unless you have small network

**Comment by vitstr** [comment]
@pevma Can i use this manual and script or could edit make script?
@yehias I have very small network :) It's a home server and home lan.

**Comment by pevma** [comment]
Yes  - you can use that script. You need to make all adjustments though.
